src/main/java/it/gov/pagopa/payment/dto/qrcode/AuthPaymentResponseDTO.java

```diff
@@ -0,0 +1,21 @@
+package it.gov.pagopa.payment.dto.qrcode;
```

Comment:
These models are details regarding the integration towards rewardCalculator, let's move in the same package
Moreover, this model is not intended to be used just for QRCode, but for all payment methods
```suggestion
package it.gov.pagopa.payment.connector.rest.reward.dto;
```